src/setops.jl

```diff
@@ -28,3 +28,77 @@ function _decompose_1D(X0::LazySet{N}) where {N}
     end
     return CartesianProductArray(out)
 end
+
+# ==============================================================
+# Construct the initial states for the continuous post-operator
+# ==============================================================
+
+using ReachabilityAnalysis: TaylorModel1, TaylorModelN, fp_rpa, zeroBox, symBox
+
+abstract type AbstractReconstructionMethod end
+
+struct ZonotopeReconstructor <: AbstractReconstructionMethod end
+
+function _reconstruct(method::ZonotopeReconstructor, P₀::LazySet, U₀::LazySet, X, ti)
+    Q₀ = P₀ × U₀
+    return Q₀
+end
+
+function _reconstruct(method::ZonotopeReconstructor, P₀::LazySet, U₀::Vector{<:LazySet}, X, ti)
+    @assert length(U₀) == 1 "expected the length of U₀ to be 1, got $(lenght(U₀))"
+    return _reconstruct(method, P₀, first(U₀), X, ti)
+end
```

Comment:
I suggest that we remove this method (later). It is probably a remainder of an early version.

Reply:
I created an issue (#177).